Last-Update: 2012-09-13
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: needs-forwarding
Description: validate .desktop file
 desktop-file-validate identified the following problems:
 * tupi.desktop: error: (will be fatal in the future): value "tupi.png"
   for key "Icon" in group "Desktop Entry" is an icon name with an extension,
   but there should be no extension as described in the Icon Theme
   Specification if the value is not an absolute path
 * tupi.desktop: warning: value "Application;Graphics;2DGraphics;RasterGraphics;"
   for key "Categories" in group "Desktop Entry" contains a deprecated value
   "Application"

--- a/qonf/configure.rb
+++ b/qonf/configure.rb
@@ -230,10 +230,10 @@
         newfile += "Name[pt]=Tupí: Magia 2D\n"
         newfile += "Name[ru]=Tupi: 2D Magic\n"
         newfile += "Exec=" + launcher_bindir + "/tupi\n"
-        newfile += "Icon=tupi.png\n"
+        newfile += "Icon=tupi\n"
         newfile += "Type=Application\n"
         newfile += "MimeType=application/tup;application/ntup;\n"
-        newfile += "Categories=Application;Graphics;2DGraphics;RasterGraphics;\n"
+        newfile += "Categories=Graphics;2DGraphics;RasterGraphics;\n"
         newfile += "Comment=2D Animation Toolkit\n"
         newfile += "Comment[es]=Herramienta para Animación 2D\n"
         newfile += "Comment[pt]=Ferramenta de animação 2D\n"
